### PR TITLE
Show multi-job output variables for deployments

### LINKED ISSUE #4946: Set a multi-job output variable does not work for deployments
Deployment is a kind of job but the Set a multi-job output variable example does not work with a deployment.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd7e0bd3-1f7d-d7b6-cc72-5ef63c31b46a
* Version Independent ID: dae87abd-b73d-9120-bcdb-6097d4b40f2a
* Content: [Variables - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch&viewFallbackFrom=vsts#set-an-output-multi-job-variable)
* Content Source: [docs/pipelines/process/variables.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/process/variables.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

### PR DESCRIPTION
**Pull request by Victor-Savu**
Show how to output variables from deployment jobs and how to later access them from another job.

Fixes #4946

### CODE REVIEW

**Comment by Slooz** [comment]
I tried implementing this and it doesn't seem to work. See https://github.com/MicrosoftDocs/vsts-docs/issues/4946#issuecomment-543319731.

**Comment by Victor-Savu** [comment]
@Slooz Sorry for the deceiving explanation I gave initially. I got confused myself by the fact that my deployment job was also named `deploy`.

**Comment by Slooz** [comment]
No worries! I tested this and am now confirming that this works for me. This method allows me to use a variable from one deployment job in another deployment job.

**Comment by PRMerger9** [comment]
@Victor-Savu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/process/variables.md [hunk 1]
```diff
@@ -364,6 +364,36 @@ jobs:
...
+The output variables of a [deployment](deployment-jobs.md) job also need to be prefixed with the job name (in this case, `A`):
+
+```yaml
+jobs:
+
+# Set an output variable from a deployment
+- deployment: A
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
```
Comment (juliakm) [inline docs/pipelines/process/variables.md]: Thanks @Victor-Savu! I found that I had to define an environment here for my YAML to successfully run. The environment should have with the same indent level as strategy and be directly above strategy. Can you add it?

Screen Shot 2019-11-08 at 10 38 20 AM

**Comment by PRMerger20** [comment]
@Victor-Savu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/process/variables.md (hunk 1 above)
Comment (Victor-Savu) [reply docs/pipelines/process/variables.md]: Thank you for the review, @juliakm! I added the environment field. Is it OK that I named it `staging`?

Comment (juliakm) [reply docs/pipelines/process/variables.md]: Thanks @Victor-Savu. That's great!

**Comment by juliakm** [comment]
#sign-off

**Pull request merged**